generators/docker-compose/templates/_consul.yml

```diff
@@ -6,13 +6,18 @@ services:
             - 8300:8300
             - 8500:8500
             - 8600:8600
-        command: consul agent -dev -ui -client 0.0.0.0
+        volumes:
+            - ./consul-conf:/consul-conf
+        command: consul agent -dev -ui -client 0.0.0.0 -config-dir /consul-conf
 
     consul-config-loader:
-        image: pbesson/consul-config-loader
+        image: jhipster/consul-config-loader
```

Comment:
Please add the v0.1.0 tag to this image. This must be done before the next release.